src/ui/effectsEngine.js

```diff
@@ -535,8 +536,9 @@ var Effect = Class('Effect', function () {
       }
     }
 
+    var count = performanceMetric.getParticleCount(data.count, opts.performance);
```

Comment:
Same change as mentioned in `ParticleEngine` note - put the properties on the `opts` object instead of a nested `performance` object